### PR TITLE
Atom/movable code cleanup

### PR DESCRIPTION
**Pull request by DominikPanic**
Cleans up unused or bad procs
Refactors some stuff to /tg/ standard

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -30,21 +28,27 @@
...
-	loc = null //so we move into null space. Must be after ..() b/c atom's Dispose handles deleting our lighting stuff
+
+	for(var/atom/movable/AM in contents)
+		qdel(AM)
+
+	moveToNullspace()
+	invisibility = INVISIBILITY_ABSTRACT
+
+	if(pulledby)
+		pulledby.stop_pulling()
```
Comment (Rohesie) [inline code/game/atoms_movable.dm]:
```suggestion
	pulledby?.stop_pulling()
```
And remove the check.

**Comment by Rohesie** [comment]
The removal of the var-edit for the mulebot is positive, though not sure if @spookydonut will mind with the map freeze.

**Comment by DominikPanic** [comment]
I can just PR it on top of his branch

**Comment by spookydonut** [comment]
> 
> 
> The removal of the var-edit for the mulebot is positive, though not sure if @spookydonut will mind with the map freeze.

one line changes like this wont conflict

**Pull request merged**